src/components/Donate/DonateDialog.js

```diff
@@ -4,47 +4,83 @@ import "@ui5/webcomponents/dist/Dialog";
 import "@ui5/webcomponents/dist/MessageStrip";
 
 import "./DonateDialog.css";
+import { BLOOD_BANK_API } from './../../app-config'
 
 class DonateDialog extends Component {
 
   constructor(props) {
     super(props);
 
     this.state = {
-
+      selectedHospital: '',
+      hospitals: []
     };
 
     this.dialogRef = React.createRef();
+    this.comboboxRef = React.createRef();
     this.createButtonRef = React.createRef();
     this.cancelButtonRef = React.createRef();
 
     this.create = this.create.bind(this);
     this.close = this.close.bind(this);
+    this.onComboboxChange = this.onComboboxChange.bind(this);
   }
 
   componentDidMount() {
     if (this.dialogRef.current) {
       this.createButtonRef.current.addEventListener("click", this.create);
       this.cancelButtonRef.current.addEventListener("click", this.close);
     }
+
+    this.comboboxRef.current.addEventListener('change', this.onComboboxChange);
```

Comment:
Add check:
if (this.comboboxRef.current) {....}

Reply:
Fixed